src/server/engagement/infra/repositories/apiEngagement.repository.ts

```diff
@@ -79,12 +79,15 @@ export class ApiEngagementRepository implements EngagementRepository {
     }
   }
 
-  buildParamètresRecherche(missionEngagementFiltre: MissionEngagementFiltre): string {
-    const { from, domain, publisher, size } = missionEngagementFiltre;
+  private buildParamètresRecherche(missionEngagementFiltre: MissionEngagementFiltre): string {
+    const { from, domain, publisher, size, lon, lat, distance } = missionEngagementFiltre;
     // eslint-disable-next-line
     const queryList: Record<string, any > = {
```

Comment:
`any` est à éviter, c'est trop large
Pour moi on manipule des `string` et des `number` donc pour moi tu peux mettre : 
`const queryList: Record<string, string | number | undefined >`

Reply:
ça marche po :( l'erreur : TS2345: Argument of type 'Record<string, string | number | undefined>' is not assignable to parameter of type 'string | Record<string, string> | string[][] | URLSearchParams | undefined'.   Type 'Record<string, string | number | undefined>' is missing the following properties from type 'string[][]': length, pop, push, concat, and 29 more.

Par contre je pense que ça rentre bien dans des discussions en jardinage nan ?